balboa-client-dispatcher/src/main/scala/com/socrata/balboa/config/DispatcherConfig.scala

```diff
@@ -0,0 +1,33 @@
+package com.socrata.balboa.config
+
+import com.socrata.balboa.config.ClientType.ClientType
+import com.socrata.balboa.metrics.config.{Configuration, Keys}
+import org.slf4j.LoggerFactory
+
+/**
+ * All inclusive Configuration file
+ */
+object DispatcherConfig {
+
+  private val Log = LoggerFactory.getLogger("DispatcherConfig")
+
+  /**
+   * @return Client Types parsed from configuration.
+   */
+  def clientTypes: Array[ClientType] = {
+    val clientStrings: Array[String] = Configuration.get().getString(Keys.DISPATCHER_CLIENT_TYPES).split(',')
```

Comment:
Add a getList functionality and move this splitting logic into there or change Configuration to be backed by Apache Commons configuration parsing which will handle lists automatically: https://commons.apache.org/proper/commons-configuration/userguide/howto_properties.html